Britannia/events/BTWK_culture_events.txt

```diff
@@ -12,30 +12,34 @@ namespace = culture_event
 ##################################################
 
 #Old Frisian -> Frisian
-#	
+	
 culture_event.001 = { #actual effect
 	type = character_event
 	hidden = true
 	
 	trigger = {
-		this = { #fired via the primate - he's always around
-			any_held_title = { title:k_papal_state }
-		}
+		has_title = title:k_papal_state 		#fired via the primate - he's always around
 		year > 650
 		NOT = { 
-			culture:frisian = { has_name_list = name_list_frisian }
+			culture:frisian = { 
+				has_name_list = name_list_frisian 
+			}
 		}
 	}	
 
-	immediate = { #Switch name list
-		culture:frisian = { set_name_list = name_list_frisian }
+	immediate = {
+		culture:frisian = {
```

Comment:
You probably want something like `every_culture_global = { limit = { has_name_list = name_list_oldfrisian } set_name_list = name_list_frisian }` to account for things like divergences and hybridizations.

Reply:
Fixed